app/Post.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
* Change author to user
* May I ask why do you need the methods getAuthorName and setAuthorName??
  * You could do $post->user->setName($name) and $post->user->getName()

Reply:
Actually it doesn't work. I don't know why. I used other methods. About author to user... Really?

Reply:
Yes. The owner of that method is Author/User. Not Post. It should work somehow

Reply:
It worked. It was strange. When I tried, it didn't work. Maybe it was because of the database or something like that. Who knows?.

Reply:
Actually, the way we are doing this is back-depending.